.golangci.yml

```diff
@@ -0,0 +1,5 @@
+issues:
+  exclude-rules:
+    - text: "SA1019: codec.LegacyAmino is deprecated"
+      linters:
+        - staticcheck
```

Comment:
From cosmos-sdk v0.42.6, the golangci-lint returns the following error:
```
app/params/encoding.go:15:21: SA1019: codec.LegacyAmino is deprecated: LegacyAmino defines a wrapper for an Amino codec that properly handles protobuf types with Any's (staticcheck)
        Amino             *codec.LegacyAmino
                           ^
x/aol/types/codec.go:10:25: SA1019: codec.LegacyAmino is deprecated: LegacyAmino defines a wrapper for an Amino codec that properly handles protobuf types with Any's (staticcheck)
func RegisterCodec(cdc *codec.LegacyAmino) {
                        ^
x/aol/keeper/query.go:14:45: SA1019: codec.LegacyAmino is deprecated: LegacyAmino defines a wrapper for an Amino codec that properly handles protobuf types with Any's (staticcheck)
func NewQuerier(k Keeper, legacyQuerierCdc *codec.LegacyAmino) sdk.Querier {
                                            ^
```
So, this check must be ignored until Amino is faded-out completely.
The reference of this yaml file is https://github.com/cosmos/cosmos-sdk/blob/a7e19ad4588bcceeeeab522f5807b9ff22e91a8c/.golangci.yml#L59